sycl/test-e2e/WorkGroupScratchMemory/dynamic_alloc_local_accessor.cpp

```diff
@@ -2,7 +2,7 @@
 // RUN: %{run} %t.out
 //
 
-// UNSUPPORTED: gpu-intel-gen12
+// UNSUPPORTED: gpu-intel-gen12, cpu
```

Comment:
https://github.com/intel/llvm/issues/16072 mentions only hip and gen12. Could you please update with information about cpu device or create a new issue if problem is different for cpu.

Reply:
fixed